packages/trader/src/Modules/Trading/Components/Form/TradeParams/Duration/simple-duration.tsx

```diff
@@ -54,7 +54,7 @@ const SimpleDuration = observer(
 
         return (
             <>
-                {duration_units_list.length > 1 && (
+                {filterMinutesAndTicks(duration_units_list).length > 1 && (
```

Comment:
Since it's used in two places, can we set the `filterMinutesAndTicks(duration_units_list)` to `filteredMinutesAndTicksList` and reuse it?

```
const filteredMinutesAndTicksList = filterMinutesAndTicks(duration_units_list);
```